packages/aws-cdk-lib/aws-ecs/lib/log-drivers/aws-log-driver.ts

```diff
@@ -106,6 +116,10 @@ export class AwsLogDriver extends LogDriver {
     if (props.logGroup && props.logRetention) {
       throw new Error('Cannot specify both `logGroup` and `logRetentionDays`.');
     }
+
+    if (props.maxBufferSize !== '' && props.mode === AwsLogDriverMode.BLOCKING) {
+      throw new Error('maxBufferSize is not valid with blocking mode');
```

Comment:
Nit: Could we make this consistent with the other error message? Something like:

```ts
Cannot specify `maxBufferSize` when the driver mode is blocking
```

Reply:
Updated